docs/new-user.rst

```diff
@@ -0,0 +1,20 @@
+New Alexa developer
+==============================================
+
+If the skills development for alexa is a new thing for you, we have some suggestion to get you deep into this world.
+
+AWS Developer account
+---------------------
+
+The first thing you need to do is configure your AWS developer account, `here <http://docs.aws.amazon.com/lambda/latest/dg/setting-up.html>`_ you can see the instructions step by step
+
+
+AWS Lambda function for a custom skill
+--------------------------------------
+
+You need to create a lambda function in order to have your skill up and running, `here you can see how <https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/developing-an-alexa-skill-as-a-lambda-function>`_ 
```

Comment:
Actually you don't, we tend to do all our skill development using ngrok and a local server. You only need a lambda when you're about to publish it. Also, you `could` not use lambda at all, and host your skill in EC2